openmetadata-spec/src/main/resources/json/schema/entity/data/dashboard.json

```diff
@@ -141,6 +141,12 @@
     "lifeCycle": {
       "description": "Life Cycle properties of the entity",
       "$ref": "../../type/lifeCycle.json"
+    },
+    "sourceHash": {
+      "description": "Source hash of the entity",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 64
```

Comment:
16 bytes might be good enough?

Reply:
@harshach I'm currently using sha256 to generate the hash of 64 char length. I'll use md5 instead which uses 32 chars. Let me know if that sounds ok